frontend/app/src/App.vue

```diff
@@ -1,41 +1,82 @@
 <template>
   <v-app id="rotki">
-    <div v-show="logged">
+    <div show="logged" class="app__content rotkiLightGrey">
       <v-navigation-drawer
         v-model="drawer"
         width="300"
+        class="app__navigation-drawer"
         fixed
-        :clipped="$vuetify.breakpoint.mdAndUp"
-        class="grey lighten-4"
+        :mini-variant="mini"
+        disable-route-watcherz
+        clipped
         app
       >
-        <div class="text-center">
-          <img
-            id="rotkehlchen_no_text"
-            :src="require('./assets/images/rotkehlchen_no_text.png')"
-            class="rotki__logo"
-            alt=""
-            @click="openSite()"
-          />
+        <div
+          v-if="!mini"
+          class="text-center rotki__logo"
+          style="
+            background-repeat: no-repeat;
+            background-position: center;
```

Comment:
What have we said about inlining styles in the templates? :D

Reply:
I had fixed this. Something weird happened when I had to rebase and correct. Ugh.

Reply:
Fixed